Resources/public/js/views/dashboard/plugins/ez-dashboardblockbaseviewplugin.js

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-dashboardblockbaseviewplugin', function (Y) {
+    'use strict';
+
+    /**
+     * Provides the dashboard base block view plugin class
+     *
+     * @module ez-dashboardblockbaseviewplugin
+     */
+    Y.namespace('eZ.Plugin');
+
+    /**
+     * The dashboard base block view plugin
+     *
+     * @namespace eZ.Plugin
+     * @class DashboardBlockBaseView
+     * @constructor
+     * @extends Base
+     */
+    Y.eZ.Plugin.DashboardBlockBaseView = Y.Base.create('baseDashboardBlockViewPlugin', Y.Base, [], {
```

Comment:
you got me wrong it seems, this plugin is outside of the scope of EZP-25760 so please remov it from  the PR